.github/workflows/frontend-ci.yml

```
@@ -144,6 +144,19 @@ jobs:
         if: ${{ env.is-pull-request == 'false' }}
         run: yarn build --max-parallel=$NX_MAX_PARALLEL
 
+  sonarcloud:
+    name: SonarCloud
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
+      - name: SonarCloud Scan
+        uses: SonarSource/sonarcloud-github-action@master
```

suggestion: Use a version range tag instead of `latest` to prevent accidental breakage
```suggestion
        uses: SonarSource/sonarcloud-github-action@v1.6
```